rust_src/src/vectors.rs

```diff
@@ -94,6 +98,31 @@ impl LispObject {
     }
 }
 
+impl Debug for LispVectorlikeRef {
+    fn fmt(&self, f: &mut Formatter) -> Result<(), fmt::Error> {
+        if let Some(v) = self.as_vector() {
+            let mut first = true;
+            write!(f, "[")?;
+            for el in v.as_slice() {
```

Comment:
Is there a better way to do this? Maybe `v.as_slice()[1:]` or something like that? Toggles are ugly.

Reply:
Hmm, I think I have a solution. Thanks for the prodding @nickdrozd.